core/src/test/java/com/klaytn/caver/feature/KlayWalletUtilsTest.java

```diff
@@ -20,9 +20,11 @@
 import com.klaytn.caver.wallet.KlayWalletUtils;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.web3j.crypto.Keys;
 import org.web3j.utils.Numeric;
 
 import java.io.File;
+import java.util.Random;
```

Comment:
Why this package is imported? I don't see the usage.

Reply:
It is obviously a mistake. I used it to make the address random, but the code didn't need to change.